.github/workflows/master.yml

```diff
@@ -0,0 +1,21 @@
+name: Master
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    name: publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Login to Docker Container Registry
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: ${{ secrets.REGISTRY_USERNAME }}/raspi-alpine-builder/raspi-alpine-builder
+          username: ${{ secrets.REGISTRY_USERNAME }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+          registry: ${{ secrets.REGISTRY_HOST }}
```

Comment:
I am fine with adding some configuration for github actions but I think it would be better in a separate merge request.

Reply:
Yup! This wasn't supposed to be a part of this Commit. I'll make sure to exclude it =)